src/test/java/org/toradocu/accuracy/PrecisionRecallCommonsCollections4.java

```diff
@@ -33,7 +33,7 @@ public void testClosureUtils() throws Exception {
 
   @Test
   public void testCollectionUtils() throws Exception {
-    test("org.apache.commons.collections4.CollectionUtils", 1, 0.935, 1, 1, 0.5, 0.25);
+    test("org.apache.commons.collections4.CollectionUtils", 1, 0.935, 1, 1, 0.428, 0.25);
```

Comment:
Do we know why precision on `@return` comments decreases?

Reply:
The imprecise handling of HTML tags was masking a false positive that is no longer masked by the improved handling of HTML tags.